client/components/shared/NewBankForm.js

```diff
@@ -128,43 +128,55 @@ export default class NewBankForm extends React.Component {
-                        { $t('client.settings.bank') }
-                    </label>
-                    <select className="form-control" id="bank" ref="bank"
-                      onChange={ this.onChangedBank.bind(this) }>
-                        { options }
-                    </select>
-                </div>
+                <form>
+                    <div className="form-group">
+                        <label htmlFor="bank">
+                            { $t('client.settings.bank') }
+                        </label>
+                        <select className="form-control" id="bank" ref="bank"
+                          onChange={ this.onChangedBank.bind(this) }>
+                            { options }
+                        </select>
+                    </div>
 
-                <div className="form-group">
-                    <label htmlFor="id">
-                        { $t('client.settings.login') }
-                    </label>
-                    <input type="text" className="form-control" id="id"
-                      ref="id"
-                      onKeyUp={ this.onKeyUp.bind(this) }
-                    />
-                </div>
+                    <div className="form-group">
+                        <div className="row">
+                            <div className="col-sm-6">
+                                <label htmlFor="id">
+                                    { $t('client.settings.login') }
+                                </label>
+                                <input type="text" className="form-control" id="id"
+                                  ref="id"
+                                  onKeyUp={ this.onKeyUp.bind(this) }
+                                />
+                            </div>
+
+                            <div className="col-sm-6">
+                                <label htmlFor="password">
+                                    { $t('client.settings.password') }
+                                </label>
+                                <input type="password" className="form-control" id="password"
+                                  ref="password"
+                                  onKeyUp={ this.onKeyUp.bind(this) }
+                                />
+                            </div>
+                        </div>
+                    </div>
 
-                <div className="form-group">
-                    <label htmlFor="password">
-                        { $t('client.settings.password') }
-                    </label>
-                    <input type="password" className="form-control" id="password"
-                      ref="password"
-                      onKeyUp={ this.onKeyUp.bind(this) }
-                    />
-                </div>
+                    { maybeCustomFields }
 
-                { maybeCustomFields }
+                    <div className="btn-toolbar pull-right">
+                        <button type="reset" className="btn btn-default">
```

Comment:
See how the bank is reset to AlloResto, which has no custom fields, but the custom field remains, although AlloResto doesn't have any? This is what I was talking about :-)